tst/SignalingApiFunctionalityTest.cpp

```diff
@@ -1445,8 +1445,8 @@ TEST_F(SignalingApiFunctionalityTest, iceRefreshEmulationWithFaultInjectionError
     BYTE firstByte = pSignalingClient->pAwsCredentials->secretKey[0];
     pSignalingClient->pAwsCredentials->secretKey[0] = 'A';
 
-    // Wait for ICE refresh while connected
-    THREAD_SLEEP(15 * HUNDREDS_OF_NANOS_IN_A_SECOND);
+    // Wait for ICE refresh while connected and terminate with an error that will be called via error callback
+    THREAD_SLEEP(30 * HUNDREDS_OF_NANOS_IN_A_SECOND);
```

Comment:
Any reason for this change?

Reply:
It seems that in some of our runs the timing is not enough to kick off the ice refresh (in 10 seconds) and also ensure it fails after it retries numerous times.